v1/tasks/reflect.go

```diff
@@ -275,15 +275,26 @@ func getIntValue(theType string, value interface{}) (int64, error) {
 		if !ok {
 			return 0, typeConversionError(value, typesMap[theType].String())
 		}
-
+		
 		return n.Int64()
 	}
-
-	n, ok := value.(int64)
-	if !ok {
+	
+	var n int64
+	switch val := value.(type) {
+	case int:
+		n = int64(val)
+	case int8:
+		n = int64(val)
+	case int16:
+		n = int64(val)
+	case int32:
+		n = int64(val)
```

Comment:
consider collapsing to single case

`case int, int8, int16, int32`

Reply:
Cannot convert an expression of the type 'interface{}' to the type 'int64'

Reply:
I noticed this PR and I think it's ready to be merged. what's the progress in present?   

> Cannot convert an expression of the type 'interface{}' to the type 'int64'
this error could be deal like this:
`	switch val := value.(type) {
	case int, int8, int16, int32:
		return int64(val.(int))
         }
`
please take a look .
@RoshanDev  @ggodik